Assets/CoreEngine/Controller/SpriteObservers/WallSpriteObserver.cs

```diff
@@ -104,6 +105,7 @@ public class WallSpriteObserver : IWallObserver, IOrientationObserver {
 		}
 
 		public void BindLevel(Level level) {
+            this.level = level;
```

Comment:
Is the indentation wrong here or github is just showing it wrong?

Reply:
Github is just showing it wrong